config/application.yml.default

```diff
@@ -243,6 +243,8 @@ phone_recaptcha_country_score_overrides: '{"AS":0.0,"GU":0.0,"MP":0.0,"PR":0.0,"
 phone_setups_per_ip_limit: 25
 phone_setups_per_ip_period: 300
 phone_setups_per_ip_track_only_mode: false
+phone_submissions_per_fingerprint_limit: 20
+phone_submissions_per_fingerprint_max_attempts_window_in_minutes: 1_440
```

Comment:
Are we planning to adjust this attempt window based on discussion in Slack / standup?

Reply:
Yes. The 10 min cooldown that already exists for SMS per user is appropriate to apply to per phone number.